pywbemcli/_cmd_instance.py

```diff
@@ -40,7 +40,7 @@
 # on instances but True on classes
 includequalifiers_option = [              # pylint: disable=invalid-name
     click.option('-q', '--includequalifiers', is_flag=True, required=False,
-                 help='Include qualifiers in the result.')]
+                 help='Include qualifiers in the returned objects.')]
```

Comment:
Now that I understand the difference between how we default the inclusion of qualifiers in returned classes vs. returned instances, I suggest that we call out "class" and "instance" specifically in the description. I.e. change the description of this instance-level option to say "returned instances" instead of "returned objects".

Also, to be consistent with the wording of the description for classes, we may want to add a sentence about the default.

Reply:
I added the preclause 'If set' to all of these options and clarified a number of them. Changed places where we used returned object to say returned instances or returned classes.